### PR TITLE
added Amber80 Solder & VCL65 Solder 

### PR DESCRIPTION
**Pull request by MaiTheSan**
Adds support for Amber80 Solder & VCL65 Solder

## Description
Add sawnsprojects/Amber80 Solder keyboard
Add sawnsprojects/vcl65 Solder keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sawnsprojects/vcl65/solder/info.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+    "keyboard_name": "VCL65 Solder",
+    "url": "",
+    "maintainer": "MaiTheSan",
+    "layouts": {
+        "LAYOUT": {
```
Comment (zvecr) [inline keyboards/sawnsprojects/vcl65/solder/info.json:6]:
```suggestion
        "LAYOUT_all": {
```

**Review by zvecr — CHANGES_REQUESTED** [review]
Also, `keyboards/sawnsprojects/amber80/solder/keymaps/multilayer/` should also be deleted as it does not compile and offeres no value over the via keymap.

**Comment by MaiTheSan** [comment]
got it

File: keyboards/sawnsprojects/amber80/solder/rules.mk [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE  = yes        # Console for debug
+COMMAND_ENABLE  = yes        # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/sawnsprojects/amber80/solder/rules.mk:16]:
```suggestion
NKRO_ENABLE = no            # Enable N-key Rollover
```

File: keyboards/sawnsprojects/vcl65/solder/rules.mk [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE  = yes        # Console for debug
+COMMAND_ENABLE  = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/sawnsprojects/vcl65/solder/rules.mk:17]:
```suggestion
```

File: keyboards/sawnsprojects/vcl65/solder/rules.mk [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/sawnsprojects/vcl65/solder/rules.mk:18]:
```suggestion
NKRO_ENABLE = no            # Enable N-key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/sawnsprojects/amber80/solder/readme.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Amber80 solder
+
+![Amber80](https://media.discordapp.net/attachments/860603126151512084/933588264047247400/IMG_0215.jpg)
```
Comment (fauxpark) [inline keyboards/sawnsprojects/amber80/solder/readme.md:3]: Could you please rehost this on imgur?

File: keyboards/sawnsprojects/amber80/solder/rules.mk [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE  = yes        # Console for debug
+COMMAND_ENABLE  = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+
```
Comment (fauxpark) [inline keyboards/sawnsprojects/amber80/solder/rules.mk:18]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/sawnsprojects/vcl65/solder/rules.mk [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE  = yes        # Console for debug
+COMMAND_ENABLE  = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+
```
Comment (fauxpark) [inline keyboards/sawnsprojects/vcl65/solder/rules.mk:18]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/sawnsprojects/vcl65/solder/readme.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# VCL65 solder
+
+![VCL65 solder](https://i.imgur.com/ofSfOTf.jpg)
```
Comment (fauxpark) [inline keyboards/sawnsprojects/vcl65/solder/readme.md:3]:
```suggestion
![VCL65 solder](https://i.imgur.com/ofSfOTfh.jpg)
```

File: keyboards/sawnsprojects/amber80/solder/readme.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# Amber80 solder
+
+![Amber80](https://i.imgur.com/ygUgpqZ.jpg)
```
Comment (fauxpark) [inline keyboards/sawnsprojects/amber80/solder/readme.md:3]:
```suggestion
![Amber80](https://i.imgur.com/ygUgpqZh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/sawnsprojects/vcl65/solder/keymaps/sawns/keymap.c [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS  
+    ),
+};
+
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
```
Comment (drashna) [inline keyboards/sawnsprojects/vcl65/solder/keymaps/sawns/keymap.c:53]: Encoders don't appear to be enabled anywhere?

Comment (MaiTheSan) [reply keyboards/sawnsprojects/vcl65/solder/keymaps/sawns/keymap.c:53]: i'm forgor

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**